### DISCUSSION TITLE
how to know knowledgeBase has saved file successfully or not?

### DISCUSSION THREAD

**Discussion by tejasso123** [agno-agi/agno#1609] [Q&A]
How can I check if a file has been successfully saved after using **knowledge_base.load(upsert=True)**?

I attempted to use a **try...except** block to catch any errors during the **load** operation, but despite seeing an error log in the console, the program does not enter the **except** block. How can I properly handle and verify if the file is saved correctly?

**Comment by dirkbrnd** [comment]
Hi
We have `exists` on the knowledge base to check whether the `vector_db` has been created and exists, but not whether anything has been loaded. So not really what you need. I agree this is not the best experience, we'll work on it.
